### PR TITLE
Add new Pages, Implement Search, and improve existing pages

### PR DESCRIPTION
**Pull request by seeta-ramaraju**
This PR includes the following changes:

Implement Search functionality
Add Panel page
Add LocusGeneDisease page
Update Gene page
Update About Project page
Update HeaderWithoutSearch component
Minor change in App component

### CODE REVIEW

**Review by olaaustine — COMMENTED** [review]
Hi @seeta-ramaraju, 
Thank you for this, the pages are looking nice. 
Just some minor suggestions. 
Thank you 
Ola

File: src/views/PanelView.vue [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+                </router-link>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <p v-else style="color: grey">Not Available</p>
+      </div>
+      <h4 class="py-3">Curators</h4>
+      <div class="row mx-3">
+        <ul v-if="panelData.curators && panelData.curators.length > 0">
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/views/PanelView.vue:301]: will take care of this in future PR

File: src/views/PanelView.vue [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        <table
+          class="table table-hover table-bordered"
+          v-if="
+            panelSummaryData.records_summary &&
+            panelSummaryData.records_summary.length > 0
+          "
+        >
+          <thead>
+            <tr>
+              <th style="width: 8%">Gene</th>
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/views/PanelView.vue]: taken care

File: src/views/LocusGeneDiseaseView.vue [hunk 3]
```diff
@@ -0,0 +1,526 @@
...
+          </tr>
+          <tr class="align-middle">
+            <td class="w-25 text-end">
+              <h4>Variant Information</h4>
+            </td>
+            <td></td>
+          </tr>
+          <tr>
+            <td class="w-25 text-end">
+              <h6>Variant Types</h6>
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/views/LocusGeneDiseaseView.vue:115]: will take care of this in future PR

File: src/views/LocusGeneDiseaseView.vue [hunk 4]
```diff
@@ -0,0 +1,526 @@
...
+              <p v-if="locusGeneDiseaseData.disease.name">
+                {{ locusGeneDiseaseData.disease.name }}
+              </p>
+              <p v-else class="text-muted">Not Available</p>
+            </td>
+          </tr>
+
+          <tr class="align-middle">
+            <td class="w-25 text-end">
+              <h6>MONDO ID</h6>
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/views/LocusGeneDiseaseView.vue]: taken care

File: src/views/LocusGeneDiseaseView.vue [hunk 5]
```diff
@@ -0,0 +1,526 @@
...
+                <tbody>
+                  <tr v-for="item in locusGeneDiseaseData.publications">
+                    <td style="width: 10%">
+                      <a
+                        v-bind:href="`https://europepmc.org/article/MED/${item.publication.pmid}`"
+                        style="text-decoration: none"
+                        v-if="item.publication.pmid"
+                      >
+                        {{ item.publication.pmid }}
+                      </a>
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/views/LocusGeneDiseaseView.vue:310]: taken care

File: src/views/LocusGeneDiseaseView.vue [hunk 6]
```diff
@@ -0,0 +1,526 @@
...
+                </tbody>
+              </table>
+              <p v-else class="text-muted">Not Available</p>
+            </td>
+          </tr>
+          <tr class="align-middle">
+            <td class="w-25 text-end">
+              <h4>
+                Confidence
+                <a href="#"><i class="bi bi-question-circle-fill"></i></a>
```
[the remark this answers is not in the document]
Comment (seeta-ramaraju) [reply src/views/LocusGeneDiseaseView.vue:261]: will take care of this in future PR

**Review by olaaustine — COMMENTED** [review]
Just one more suggestion @seeta-ramaraju. 
Thank you.

**Pull request merged**